src/helpers/jit-helper.js

```diff
@@ -707,12 +707,58 @@ class JitHelper {
    */
   static buildTmpPath(config) {
     const tmpPath = homePath(jitPath, `${config.name}_${config.project.code}`);
+    fse.ensureDirSync(tmpPath);
+
+    return tmpPath;
+  }
+
+  /**
+   * @param {String} componentPath
+   * @param {Object} data
+   * @param {Boolean} isHCL2
+   * @return {Promise}
+   * @private
+   */
+  static _saveToFile(componentPath, data, isHCL2) {
+    let formatHCL1 = "";
+    if (!isHCL2) {
+      formatHCL1 = "-F no"
+    }
+
+    const arch = Downloader.getOsArch();
+    const componentBinPath = join(binPath, arch);
+
+    let extension = '';
+    if (arch.indexOf("windows") > -1)
+      extension = '.exe';
+
+    const dataStringify = JSON.stringify(data);
+    const buff = new Buffer(dataStringify);
+    const base64data = buff.toString('base64');
+
+    return exec(`${join(componentBinPath, `converter${extension}`)} -i '${base64data}' ${formatHCL1}`)
+      .then(result => {
+        return fse.outputFileSync(componentPath, result.stdout);
+      }).catch(err => {
+        return Promise.reject(err);
+      });
+  }
 
-    if (!fse.existsSync(tmpPath)) {
-      fse.mkdirsSync(tmpPath);
+  /**
+   * @param {Object} config
+   * @return {Boolean}
+   */
+  static checkTfVersion(config) {
+    const { terraform } = config;
+    if (terraform) {
+      const { version } = terraform;
+      const regExVersion = /0\.11/gm;
```

Comment:
this implementation works only tf v0.11, but doesn't work with v0.10 or v0.9 ... please use `semver` instead, see https://github.com/TerraHubCorp/terrahub/blob/master/bin/cli.js#L6